nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/AICorpus.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * (C) Copyright 2006-2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * Contributors:
+ *     anechaev
+ */
+package org.nuxeo.ai.cloud;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AICorpus {
```

Comment:
You just renamed `AICorpus` to `DatasetExport`, can we give this a better name and some javadoc?

Reply:
e.g. POJO for a DocumentType in the cloud

Reply:
done

Reply:
it's AI_Corpus, I'm not sure what can be a better name :)